release-notes/release-notes58.md

```diff
@@ -100,37 +101,15 @@ The {{site.productname}} 5.8 release includes an accompanying release of **Tiny
 
 For information on Tiny Drive, see: [Tiny Drive]({{ site.baseurl }}/tinydrive/).
 
-## Accompanying Premium self-hosted server-side component changes
```

Comment:
Should all of this be deleted?

Reply:
Based on the information I have about what is in this release, this section will not be needed.